azure/durable_functions/models/DurableOrchestrationClient.py

```diff
@@ -74,7 +74,9 @@ async def start_new(self,
         if response[0] <= 202 and response[1]:
             return response[1]["id"]
         else:
-            return None
+            # Surfacing the durable-extension exception
+            exception_message = response[1]
+            raise Exception(exception_message)
```

Comment:
Can we add an `elif response[0] == 400:` case which tries to get the value of the `Message` field and just include that text in the Exception, like I suggested in the other thread?

Reply:
I originally decided against that because I could not **fully** control how the exception message got displayed. I believe my client-level exception is getting catched and re-thrown by the worker. The result is that the error messaged looks something like this:

```
Exception: Exception: Orchestration not found ....
```

Noticed the only exception above ^. I think this is a worker-level fix, but I need to investigate further. In any case, I'm making the SDK-level fix now, with your suggestion, and will follow-up on the double exception prefix later :) . Please see my latest commit